src/test/java/io/redis/examples/BloomFilterExample.java

```diff
@@ -0,0 +1,45 @@
+// EXAMPLE: bf_tutorial
+
+// HIDE_START
+package io.redis.examples;
+
+import redis.clients.jedis.UnifiedJedis;
+import org.junit.Test;
+import org.junit.Assert;
+import java.util.List;
+
+public class BloomFilterExample {
+
+    @Test
+    public void run() {
+        UnifiedJedis unifiedJedis = new UnifiedJedis("redis://localhost:6379");
+        // HIDE_END
+
+        // STEP_START bloom
+        String res1 = unifiedJedis.bfReserve("bikes:models", 0.01, 1000);
+        System.out.println(res1); // >>> True
+
+        boolean res2 = unifiedJedis.bfAdd("bikes:models", "Smoky Mountain Striker");
+        System.out.println(res2); // >>> True
+
+        boolean res3 = unifiedJedis.bfExists("bikes:models", "Smoky Mountain Striker");
+        System.out.println(res3); // >>> True
+
+        List<Boolean> res4 = unifiedJedis.bfMAdd("bikes:models",
+                "Rocky Mountain Racer",
+                "Cloudy City Cruiser",
+                "Windy City Wippet");
+        System.out.println(res4); // >>> True
```

Comment:
res4 is a list of booleans. So the output can't only be `True`.

Reply:
@sazzad16 fixed it 
Thank you